packages/node_modules/nav-frontend-alertstriper/src/alertstripe.js

```diff
@@ -45,12 +46,14 @@ AlertStripe.propTypes = {
     children: PT.node.isRequired,
     type: PT.oneOf(['advarsel', 'suksess', 'info']).isRequired,
     solid: PT.bool,
+    size: PT.string,
```

Comment:
At size er string her fører til følgende lint-warning på Icon-komponenten:
Warning: Failed prop type: Invalid prop `size` of type `string` supplied to `Icon`, expected `number